cattle/utils.py

```diff
@@ -87,10 +87,17 @@ def get_containers(self):
         return self._get(self.url + '/containers')
 
     def get_latest_stat(self):
-        return self.get_containers()['stats'][-1]
+        containers = self.get_stats()
+        if containers:
```

Comment:
Thats a good catch. I did some refactoring of this, and it used to call `get_containers` method directly. It is very unlikely in this state to be null.